kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/PartitionPruningSuite.scala

```diff
@@ -242,11 +241,15 @@ class PartitionPruningSuite extends AnyFunSuite with TestUtils with ExpressionTe
       (
         // Filter on just the data column
         // 1637202600123456L in epoch micros for '2021-11-18 02:30:00.123456'
-        predicate("=", col("tsNtz"), ofTimestampNtz(1637202600123456L)),
```

Comment:
Why do we have this change?

Reply:
Ah this logic worked previously because `timestamp_ntz` was data skipping ineligible Allison. It mainly validated the remainingFilter logic on data columns and I updated it to test for the same after.